common/components/ui/Identicon.tsx

```diff
@@ -24,6 +24,7 @@ export default function Identicon(props: Props) {
         <React.Fragment>
           <img
             src={identiconDataUrl}
+            alt="Unique Address Image"
```

Comment:
How much of a pain in the ass would it be for users to hear "Address image for 0x124ab21f...", because I feel like this alt text doesn't tell you what the image is of.

Reply:
I agree, it would improve the context for the user by providing the address. I'm not sure how hard that'd be but I'll look into it.